strawberry/ext/mypy_plugin.py

```diff
@@ -254,18 +256,19 @@ def enum_hook(ctx: DynamicClassDefContext) -> None:
 
 
 def add_method_to_class(
-    api: SemanticAnalyzerPluginInterface,
+    api: Union[SemanticAnalyzerPluginInterface, CheckerPluginInterface],
     cls: ClassDef,
     name: str,
     args: List[Argument],
     return_type: Type,
     self_type: Optional[Type] = None,
-    tvar_def: Optional[TypeVarDef] = None,
+    tvar_def: Optional[TypeVarType] = None,
     is_static: bool = False,
 ) -> None:
     """Adds a new method to a class definition.
-    Edited to add is_static for
-    https://github.com/python/mypy/blob/master/mypy/plugins/common.py
+    Edited to add is_static logic for
+    https://github.com/python/mypy/blob/9c05d3d19de74fc10a51aa5b663e6a38bc6abc73
+    /mypy/plugins/common.py
```

Comment:
I think it's a good adjustment regardless, but was there something that triggered the need to use a permalink here?

I think this got accidentally wrapped

Reply:
i wanted to link permanently to the version of mypy i copied from XD
and flake kept complaining about the line being too long. how do i fix it?

Reply:
I think you can put a 
```python
# noqa: E501
```
comment above it

(https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes)